examples/targeting/add_customer_negative_criteria.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Adds various types of negative criteria as exclusions at the customer level.
+
+These criteria will be applied to all campaigns for the given customer.
+"""
+
+
+import argparse
+import sys
+
+from google.ads.google_ads.client import GoogleAdsClient
+from google.ads.google_ads.errors import GoogleAdsException
+
+
+def main(client, customer_id):
+    """The main method that creates all necessary entities for the example.
+
+    Args:
+        client: an initialized GoogleAdsClient instance.
+        customer_id: a client customer ID.
+    """
+    tragedy_criterion_op = client.get_type(
+        "CustomerNegativeCriterionOperation", version="v5"
+    )
+    tragedy_criterion = tragedy_criterion_op.create
+    # Creates a negative customer criterion excluding the content label type
+    # of 'TRAGEDY'.
+    tragedy_criterion.content_label.type = client.get_type(
+        "ContentLabelTypeEnum", version="v5"
+    ).TRAGEDY
+
+    placement_criterion_op = client.get_type(
+        "CustomerNegativeCriterionOperation", version="v5"
+    )
+    placement_criterion = placement_criterion_op.create
+    # Creates a negative customer criterion excluding the placement with url
+    # 'http://www.example.com'.
+    placement_criterion.placement.url = "http://www.example.com"
+
+    customer_negative_criterion_service = client.get_service(
+        "CustomerNegativeCriterionService", version="v5"
+    )
+
+    try:
+        # Issues a mutate request to add the negative customer criteria.
+        response = customer_negative_criterion_service.mutate_customer_negative_criteria(
```

Comment:
This line exceeds the char limit, and newlining it doesn't help either. I didn't want to rename the vars so leaving as-is seemed like the most readable option.

Reply:
how about something like :
```
response = (customer_negative_criterion_service
                       .mutate_customer_negative_criteria(customer_id, 
                           [tragedy_criterion_op, placement_criterion_op]
                    ))
```

Reply:
Same as in the other PR, the auto-formatter will undo it and use a style that's longer than 88 chars.